containers/agent/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+)
+
+const (
+	componentName = "osp-director-operator-agent"
+)
+
+var (
+	rootCmd = &cobra.Command{
+		Use:   componentName,
+		Short: "Run OSP-Director-Operator Deploy Agent",
+		Long:  "Runs the OSP-Director-Operator Deploy Agent",
```

Comment:
Do we want to make this more generic now, i.e. remove `Deploy`, in anticipation of reusing this agent for more than just the deploy option?

Reply:
good call. will do

Reply:
fixed in latest commits